### PR TITLE
HA Replicas Rework

### PR DESCRIPTION
**Pull request by ggodeke**
# Description

We are approaching the launch of read replicas, and our HA Replicas docs are being rebranded/reworked a bit as a result. I've taken the existing high availability page and merged it into the HA replicas section (now just called high availability) and included some edits to our backup and restore docs.

I've also worked in the replica switchover documentation. 

I think this is complete, aside from grammar and editing magic from the help of the docs team 😄 I think the only thing missing is that: 
1. I need to adjust the screenshot of the instruction once the read replicas go live. I will submit a separate PR for this post-launch.
2. We may want to host the failover images independently. Currently, I stole the assets from the high availability blog post.
3. We may want a redirect from the old high availability URL to the HA replicas page since this includes the removal of that page.

If there is anything I can do to help make this review easier, please let me know! 🙇 

# Links

Fixes private #63 [document replica switchover](https://github.com/timescale/docs-private/issues/63)

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/timescaledb/latest/contribute-to-docs/)

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [x] Is the content technically accurate?
- [x] Is the content complete?
- [x] Is the content presented in a logical order?
- [x] Does the content use appropriate names for features and products?
- [x] Does the content provide relevant links to further information?

## Documentation team review checklist

- [ ] Is the content free from typos?
- [ ] Does the content use plain English?
- [ ] Does the content contain clear sections for concepts, tasks, and references?
- [ ] Have any images been uploaded to the correct location, and are resolvable?
- [ ] If the page index was updated, are redirects required
      and have they been implemented?
- [ ] Have you checked the built version of this content?

### CODE REVIEW

File: cloud/backup-restore-cloud.md [hunk 1]
```diff
@@ -8,104 +8,27 @@ tags: [recovery, failures]
...
+incremental backups every day. Additionally, all WAL ([Write-Ahead Log]
+[wal]) files are retained back to the oldest full backup. This means
+that you always have a full backup available for the current and
+previous week, and your service can be restored to any point during this
+time period.
+
+In the event of a storage failure, a service will automatically restore
+from backup to the point of failure. In the event of a user error where
+a point-in-time recovery needs to be done, please contact support. We do
+not currently offer point-in-time recovery through the UI.
```
Comment (Rajakavitha1) [inline cloud/backup-restore-cloud.md:25]:
```suggestion
a point-in-time recovery needs to be done, please [contact support][support]. 
```

File: cloud/backup-restore-cloud.md [hunk 2]
```diff
@@ -8,104 +8,27 @@ tags: [recovery, failures]
...
-    than the restore step.
-
-If you want to check if your service is in the recovery phase, you can run this
-query from the `psql` prompt:
-
-```sql
-SELECT pg_is_in_recovery()::text
-```
-
-[support]: https://www.timescale.com/support
```
Comment (Rajakavitha1) [inline cloud/backup-restore-cloud.md:108]: [support]: https://www.timescale.com/support

File: cloud/backup-restore-cloud.md [hunk 3]
```diff
@@ -8,104 +8,27 @@ tags: [recovery, failures]
...
+
+In the event of a storage failure, a service will automatically restore
+from backup to the point of failure. In the event of a user error where
+a point-in-time recovery needs to be done, please contact support. We do
+not currently offer point-in-time recovery through the UI.
+
+To understand how a restore from backup occurs after a storage failure,
+please see the following diagram. (Note: compute failures are able to
+avoid a full restore from backup. For more information, see our docs on
+[rapid recovery][rapid-recovery].)
```
Comment (Rajakavitha1) [inline cloud/backup-restore-cloud.md:30]:
```suggestion
[rapid recovery][rapid-recovery].) </Highlight>
```

File: cloud/backup-restore-cloud.md [hunk 4]
```diff
@@ -8,104 +8,27 @@ tags: [recovery, failures]
...
+previous week, and your service can be restored to any point during this
+time period.
+
+In the event of a storage failure, a service will automatically restore
+from backup to the point of failure. In the event of a user error where
+a point-in-time recovery needs to be done, please contact support. We do
+not currently offer point-in-time recovery through the UI.
+
+To understand how a restore from backup occurs after a storage failure,
+please see the following diagram. (Note: compute failures are able to
```
Comment (Rajakavitha1) [inline cloud/backup-restore-cloud.md:28]:
```suggestion
This diagram helps you to understand how a restore from backup occurs after a storage failure.
<Highlight type="note">compute failures are able to
```

**Review by Rajakavitha1 — CHANGES_REQUESTED** [review]

File: cloud/backup-restore-cloud.md (hunk 2 above)
Comment (ggodeke) [reply cloud/backup-restore-cloud.md:108]: hi @Rajakavitha1, thanks for the review! 🙇 Is there something you wanted me to adjust here?

File: cloud/backup-restore-cloud.md [hunk 5]
```diff
@@ -9,25 +9,24 @@ tags: [recovery, failures]
...
+In the event of a storage failure, a service automatically restores
 from backup to the point of failure. In the event of a user error where
-a point-in-time recovery needs to be done, please contact support. We do
-not currently offer point-in-time recovery through the UI.
+a point-in-time recovery needs to be done, please [contact support][support]. 
 
-To understand how a restore from backup occurs after a storage failure,
-please see the following diagram. (Note: compute failures are able to
+This diagram helps you to understand how a restore from backup occurs after a storage failure.
+<Highlight type="note">compute failures are able to
```
Comment (Loquacity) [inline cloud/backup-restore-cloud.md:27]:
```suggestion

<Highlight type="note">
Compute failures are able to
```

File: cloud/backup-restore-cloud.md [hunk 6]
```diff
@@ -9,25 +9,24 @@ tags: [recovery, failures]
...
 avoid a full restore from backup. For more information, see our docs on
```
Comment (Loquacity) [inline cloud/backup-restore-cloud.md:28]:
```suggestion
avoid a full restore from backup. For more information, see the
```

File: cloud/backup-restore-cloud.md [hunk 7]
```diff
@@ -9,25 +9,24 @@ tags: [recovery, failures]
...
-[rapid recovery][rapid-recovery].) 
+[rapid recovery][rapid-recovery].) </Highlight>
```
Comment (Loquacity) [inline cloud/backup-restore-cloud.md:29]:
```suggestion
[rapid recovery section][rapid-recovery].) 
</Highlight>
```

File: cloud/backup-restore-cloud.md (hunk 6 above)
Comment (Loquacity) [reply cloud/backup-restore-cloud.md:28]: I'm not sure I understand what this means?

File: cloud/backup-restore-cloud.md (hunk 2 above)
Comment (Rajakavitha1) [reply cloud/backup-restore-cloud.md:108]: Hi @ggodeke !!! adding the link to support might help

**Review by Rajakavitha1 — APPROVED** [review]

File: cloud/backup-restore-cloud.md (hunk 6 above)
Comment (ggodeke) [reply cloud/backup-restore-cloud.md:28]: I'll rework this -- thanks!

**Pull request merged**